app/css/ui-kit/inputs.css

```diff
@@ -20,6 +20,13 @@ textarea {
     }
 }
 
+/* maybe make this default for all inputs */
+.portable-input {
+    &::placeholder {
+        @apply text-textColor6 !important;
+    }
+}
+
```

Comment:
What does this do? (Any time there's an `!important` it's generally a smell!)

Reply:
This is the only way the change took effect. I agree on the `!important` I don't like it using too :)